project/thirdparty/mbedtls-flags.xml

```diff
@@ -0,0 +1,9 @@
+<xml>
+
+  <depend name="${HXCPP}/project/thirdparty/mbedtls-flags.xml" dateOnly="true" />
+
+  <compilerflag value="-I${HXCPP}/project/thirdparty/mbedtls-2.28.2/include" />
```

Comment:
Include path has to be updated here as well now. I think eventually we should switch to using submodules and just have a `mbedtls` directory rather than `mbedtls-x.x.x`.

Reply:
I definitely wouldn't want to see a change to submodules, they are hell on earth and it would be better if the git world forgot they existed. Switching to a sane alternative like subtrees would be good though.